purchase_sale_inter_company/models/stock_picking.py

```diff
@@ -18,7 +18,8 @@ def action_done(self):
             lambda x: x.location_dest_id.usage == "customer"
         ).sudo():
             purchase = pick.sale_id.auto_purchase_order_id
-            if not purchase:
+            # Not create from inter-company or has products tracked by lots
+            if not purchase or pick.has_tracking:
```

Comment:
```suggestion
            if not purchase:
              continue
            if pick.has_tracking:
              continue
```

Reply:
Just IMHO, I would think it is better to separated case, and keep existing as is.
As the 2 case are quite different.

Reply:
For me, I think no need for the comment, as the `pick.has_tracking` is quite clear that it has lot/serial.